packages/pharos/src/components/toast/pharos-toaster.test.ts

```diff
@@ -118,4 +138,25 @@ describe('pharos-toaster', () => {
     const toast = component.querySelector('pharos-toast');
     expect(toast).to.be.null;
   });
+
+  it('does a things', async () => {
```

Comment:
Not sure I understand this test—how does it exercise the updateability?

Reply:
I'm sorry...that was left in before I was entirely sure what I was testing.

Reply:
It tests updatability by creating a toast in `fireOpenUpdateableToastEvent` and then updates the toast in `fireUpdateToastEvent`. The test expects the updated message, not the message it was created with.

Reply:
Ah I glossed over a couple of names that your callout helped me make sense of. Got it.